pkg/kubelet/cm/memorymanager/state/state_checkpoint_test.go

```diff
@@ -89,7 +85,7 @@ func TestCheckpointStateRestore(t *testing.T) {
 				machineState: MemoryMap{
 					0: map[core.ResourceName]MemoryTable{
 						core.ResourceMemory: MemoryTable{
-							Allocatable:    1024,
+							Allocatable:    1536,
```

Comment:
In fact we could devote a separate assert to validate all those values 😄 (just an idea, no need to implement...)
The assert could check whether, e.g. `Allocatable = TotalMemSize - SystemReserved`, and so forth.

Reply:
Probably we will need such unit tests under the policy memory calculation.

Reply:
Cool.